app/views/buyers/accounts/show.html.slim

```diff
@@ -62,13 +62,13 @@ h1 data-hook="account-show"
                                  reject_admin_buyers_account_path(@account),
                                  method: :post,
                                  class: 'reject action'
-            - if can? :suspend, @account
+            - unless @account.suspended_or_scheduled_for_deletion?
```

Comment:
@damianpm @didierofrivia @guicassolato Why not `- if @account.can_suspend?` as I explained in https://github.com/3scale/porta/pull/2130#discussion_r472221448

![image](https://user-images.githubusercontent.com/11318903/91538070-8601b580-e917-11ea-91df-d221199d8304.png)

![image](https://user-images.githubusercontent.com/11318903/91538090-8dc15a00-e917-11ea-8525-e19a65bb5a7b.png)

Reply:
Because it's not the same. `#can_suspend?` may be idiomatic but will return based on the existing constraints of the event machine, which now reduces to `#ready_to_be_suspended?` (wisely [pointed](https://github.com/3scale/porta/pull/2130#discussion_r478465369) by @didierofrivia that should be named `#ready_to_suspend?` instead, BTW).

If I understood correctly, we didn't want to add any constraint to the event machine, but just decide based on the current state.

Reply:
I disagree but I will explain properly (in-depth) on Tuesday. So today we can focus on learning as intended :wink: